app/index.html

```diff
@@ -6,6 +6,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <title>crates.io: Rust Package Registry</title>
+    <meta name="description" content="The Rust community’s crate registry" />
+    <meta name="og:title" content="Rust Package Registry" />
+    <meta name="og:description" content="The Rust community’s crate registry" />
+    <meta name="og:image" content="https://crates.io/assets/Cargo-Logo-Small.png" />
+    <meta name="theme-color" content="#f9f7ec" />
```

Comment:
I just had a closer look at this and it looks like https://github.com/rust-lang/crates.io/blob/69f90830490b1840d86266de719af5520d6512a2/config/manifest.js is already adding the `theme-color` meta tag at build time.

Reply:
Sorry, I think a certain mobile broywser won't apply this special on iOS unless the website is PWA.

Sorry if I am wrong :(